src/main/java/com/epam/trn/hw6/pages/MetalsAndColors.java

```diff
@@ -0,0 +1,114 @@
+package com.epam.trn.hw6.pages;
+
+import com.epam.jdi.light.elements.complex.Checklist;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.Title;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Checkbox;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+import com.epam.trn.hw6.utils.DataSet;
+import com.epam.trn.hw6.utils.JsonDataReader;
+import org.openqa.selenium.WebElement;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Title("Metal and Colors")
+public class MetalsAndColors extends WebPage {
```

Comment:
Where is MetalsAndColors Form ?
Take a look at Login form and Entity Driving Testing approach from the lecture.
My expectations is: Class -> Page -> Form::submit(entity)